py/specs/select/count.toml

```diff
@@ -0,0 +1,21 @@
+[setup]
+statement_files = ["sql/uservisits.sql"]
```

Comment:
if the filepaths are relative to the toml file, then it should start with `../sql`